Sources/JExtractSwiftLib/SwiftTypes/SwiftEffectSpecifier.swift

```diff
@@ -0,0 +1,17 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift.org project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of Swift.org project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+enum SwiftEffectSpecifier: Equatable {
+  case `throws`
+}
```

Comment:
I think that's okey, in swift-syntax they're just as a keyword afair -- any opinion about such enum @rintaro ?

Reply:
The enum look okay to me for now. We should support typed throw some day, but not now.